enarx.ks

```diff
@@ -53,6 +53,13 @@ install -o root -g root -m 600 /dev/stdin /etc/sudoers.d/nopasswd <<EOF
 %wheel	ALL=(ALL)	NOPASSWD: ALL
 EOF
 
+# Give SGX and SEV device node access to their respective groups
+groupadd -f sgx
+echo "SUBSYSTEM==\"sgx\", MODE=\"0666\", GROUP=\"sgx\"" > /etc/udev/rules.d/50-sgx.rules
+
+groupadd -f sev
+echo "KERNEL==\"sev\", MODE=\"0666\", GROUP=\"sev\"" > /etc/udev/rules.d/50-sev.rules
```

Comment:
1. Use the kickstart `group` command to create the groups.
2. Use single quotes to avoid escaping the double quotes.
3. The mode is wrong.